sample-apps/deepedit_left_atrium/main.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, app_dir, studies):
         }
 
         resources = [
-            (self.pretrained_model, "https://www.dropbox.com/s/fgygtodqztjjdyz/deepedit_heart.pt?dl=1"),
+            (self.pretrained_model, "https://www.dropbox.com/s/fgygtodqztjjdyz/deepedit_left_atrium.pt?dl=1"),
```

Comment:
please fix this to: https://www.dropbox.com/s/kvwcudzxb4mjbcp/deepedit_left_atrium.pt?dl=1